Documentation/contributing/development/reviewers_committers/triage.rst

```diff
@@ -0,0 +1,153 @@
+issues. We want to identify serious bugs and fix them.
+
+Handle these GitHub issues: `cilium/cilium issues that need Triager attention
+<https://github.com/cilium/cilium/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds%2Ftriage+-label%3Aarea%2Fdatapath+-label%3Aarea%2Fagent+-label%3Aarea%2Fhubble+-label%3Aarea%2FCI+-label%3Aarea%2FCI-improvement+-label%3Akind%2Ffeature+-label%3Akind%2Fenhancement+-label%3Akind%2Ftask>`_
+
+Issues are removed from this filter when they have the following labels added:
+
+*   ``area/datapath``
+*   ``area/agent``
+*   ``area/hubble``
+*   ``area/CI``
+*   ``area/CI-improvement``
+*   ``kind/feature``
+
+Issues are also removed from this filter when they have the following types
+set:
+
+*   Enhancement
+*   Task
+
+Please don't make changes to the filter without discussing in the `#tophat
+<https://slack.com/app_redirect?channel=tophat>`_ channel first.
+
+The effort we spend should be proportional to the reporter's effort. This
+typically means a 5-10 minute assessment of overall impact, project area etc.
+
+If the issue isn't clear enough request for more information like a sysdump or
+steps to replicate the issue. Add label ``need-more-info``. Adding
+``need-more-info`` will remove the issue from display in the filter, until the
+original poster responds, when automation will change ``need-more-info`` to
+``info-completed``. This allows issues that are logged with not enough
+information and no followup to not waste Triager time looking at them.
+
+For GH issues that have the "Regression" section filled out or otherwise look
+like regressions, (for example: "this worked before but it's not working
+now"), bring it up immediately to the `#vertical-agent
+<https://slack.com/app_redirect?channel=vertical-agent>`_ or `#vertical-datapath
+<https://slack.com/app_redirect?channel=vertical-datapath>`_. Also add the
+``kind/regression`` label.
+
+If it has enough information but is not a regression, decide if it's
+``area/agent``, ``area/datapath`` or ``area/hubble``. When in doubt which one is
+it, bring it up in the developer meeting.
+
+Every issue in ``cilium/cilium`` should have ``area/agent``, ``area/datapath``
+or ``area/hubble`` once it has enough information, on rare occasions it can be
+both agent and datapath.
+
+If possible add ``"area/*"`` labels and more specific labels, like
+``sig/k8s``, ``area/clustermesh``, etc.
+
+If someone has created an issue outside the issue templates, make your best
+guess and ping the creator to remind them to label their issues. This will
+stop unlabeled issues from building up over time.
+
+Once you have added ``area/agent``, ``area/datapath`` or ``area/hubble`` remove
+the ``needs/triage`` label.
+
+Try to achieve an empty queue. There is no date filter, as that means
+explicitly ignoring older issues. If an issue is not relevant, it should be
+closed. If the queue grows faster than we can handle, changes need to be made
+and discussions need to be had.
```

Comment:
This last sentence is not useful. The writing is passive and implies that someone somewhere should do something. If we want a process that triagers can follow, we should describe exactly what they should do. If we don't know, we can fall back to "raise a discussion about the workload in #development" (or #triage or whatever channel we decide is good for this work).